x/concentrated-liquidity/incentives_test.go

```diff
@@ -3194,9 +3187,14 @@ func (s *KeeperTestSuite) TestClaimAllIncentives() {
 			initSenderBalances := s.App.BankKeeper.GetAllBalances(s.Ctx, defaultSender)
 			initPoolBalances := s.App.BankKeeper.GetAllBalances(s.Ctx, clPool.GetAddress())
 
+			if !tc.forfeitIncentives {
```

Comment:
Note: to avoid a larger test refactor, I kept `forfeitIncentives` as a boolean test field, where it either forfeits some or all incentives. Partial incentive forfeiting is tested in LP withdrawal tests in `lp_test.go`